Converts legacy PDF file to HTML. Fixes links. (#340)
apache/www-site: content/foundation/docs/targeted-sponsorship-policy.md

Title: Targeted Sponsorship Policy
license: https://www.apache.org/licenses/LICENSE-2.0

# {{title}}

## What is a Targeted Sponsor?

Targeted sponsors support the foundation by helping specific parts of the foundation.
For example, targeted sponsors today donate cloud services, fund project hackathons,
and provide committer resources. It’s our way of encouraging and recognizing the
sponsors that we rely on everyday outside of and often in addition to funding our general
operations.

## Who Can Be a Targeted Sponsor?

> The policy available at
> [apache.org/foundation/sponsorship.html#WhoCanSponsor](https://www.apache.org/foundation/sponsorship.html#WhoCanSponsor)

Reviewer: ```suggestion
The policy available as ["Who Can Sponsor The ASF"](/foundation/sponsorship.html#who-can-sponsor-the-asf)
```

also applies to targeted sponsors as will the same sponsor agreement
template and intake process.

## Changes to Thanks Page:

The main ASF Sponsor Thanks Page is now split into two sections, “Normal”,
non-targeted Sponsors and “Targeted” Sponsors.

The previous Infrastructure sponsorship section on the Thanks page was removed and
carefully audited with Infra to integrate Infrastructure sponsors as simply one type of
targeted sponsorship.

## More Details

Committees and projects can choose to set up a project thanks area of their TLP site
that also recognizes the sponsor in sync with the foundation page as noted at

> [apache.org/foundation/marks/linking](https://www.apache.org/foundation/marks/linking). 

Reviewer: ```suggestion
the [Apache® Project Corporate Recognition Best Practices](/foundation/marks/linking). 
```
